mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -675,6 +672,14 @@ LogicalResult getTuningProblemStr(rock::RockGemmWrapperInterface gemmIF,
     SmallString<5> iLayout("#####");
     SmallString<5> oLayout("#####");
 
+    if (size > 5) {
```

Comment:
Code below relies on the layout strings being the correct length so it can overwrite characters.  This code doesn't handle 1-D, which migraphx doesn't care about but other paths might.

Reply:
... you could instead `SmallString<8> oLayout(size, '#');` above - I might have the constructor backwards